dlp/dlp_inspect_image_file.rb

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# [START dlp_inspect_image_file]
+require "google/cloud/dlp"
+
+def inspect_image_file (
+    project_id = "YOUR_PROJECT_ID",
+    filepath   = "path/to/image.png"
+  )
+
+  # Instantiate a client
+  dlp = Google::Cloud::Dlp.new
+
+  # Get the bytes of the file
+  file_bytes = File.open(filepath, "rb").read()
```

Comment:
`file_bytes = File.read filepath`

Reply:
I don't think this captures the "b" in "rb". I tried changing it to this line and got an error:
```
/Users/ajhamilton/.rbenv/versions/2.4.4/lib/ruby/gems/2.4.0/gems/google-gax-1.0.1/lib/google/gax/util.rb:64:in `initialize': "\x89" on UTF-8 (Encoding::InvalidByteSequenceError)
```
Is there a way to use `File.read` and also specify binary file mode?

Reply:
`File.binread filepath`